app/src/main/kotlin/com/hippo/ehviewer/util/KtorExtensions.kt

```diff
@@ -0,0 +1,17 @@
+package com.hippo.ehviewer.util
+
+import io.ktor.client.call.body
+import io.ktor.client.statement.HttpResponse
+import io.ktor.utils.io.charsets.decode
+import io.ktor.utils.io.core.ByteReadPacket
+import java.nio.charset.CodingErrorAction
+
+suspend fun HttpResponse.bodyAsUtf8Text(): String {
+    val decoder = Charsets.UTF_8.newDecoder().apply {
```

Comment:
Is `CharsetDecoder` thread safe?
Consider make it a top-level val or `ThreadLocal`